stream/tests/test_client.py

```diff
@@ -152,15 +152,15 @@ def test_api_url(self):
             )
 
     def test_collections_url_default(self):
-        client = stream.connect("key", "secret")
-        feed_url = client.get_full_url(relative_url="meta/", service_name="api")
+        c = stream.connect("key", "secret")
```

Comment:
why all the renaming client to c?

Reply:
not everywhere but in some places, not mix with the global one